src/web/components/SharingPermission/SharingPermissionsTable.tsx

```diff
@@ -0,0 +1,86 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState } from 'react';
+
+import { ParticipantPayload } from '../../services/participant';
+import { ParticipantsTable } from './ParticipantsTable';
+
+import './SharingPermissionsTable.scss';
+
+type SharingPermissionsTableProps = {
+  sharedParticipants: ParticipantPayload[];
+};
+
+function NoParticipant() {
+  return (
+    <div className='no-participants-container'>
+      <img src='/group-icon.svg' alt='group-icon' />
+      <div className='no-participants-text'>
+        <h1>No Participants</h1>
+        <span>You don&apos;t have any sharing permissions yet.</span>
+      </div>
+    </div>
+  );
+}
+export function SharingPermissionsTable({ sharedParticipants }: SharingPermissionsTableProps) {
+  const [filter, setFilter] = useState('');
```

Comment:
should we call this filterText to make it explicitly a noun ?

Reply:
what do you mean by explicitly a noun?

Reply:
The word filter is typically also used for a action/operation - like `.filter( x -> .. predicate )`
Here, it seems this is the text that needs to be used for a future filter operation - so adding the suffix would make it explicit and easier to read imo.